### ISSUE TITLE
[CLI]: Wandb 0.17.6 breaks lightgbm integration

### ISSUE THREAD

**Issue by skaae** [wandb/wandb#8157]
### Describe the bug

When using Wandb 0.17.6 and lightgbm 4.5.0 we get an attribute error in the lightgbm integration. We do not get the error in Wandb 0.17.5.

```
.venv/lib/python3.10/site-packages/wandb/integration/lightgbm/__init__.py:238: in log_summary
    with wb_telemetry.context() as tel:
.venv/lib/python3.10/site-packages/wandb/sdk/lib/telemetry.py:42: in __exit__
    self._run._telemetry_callback(self._obj)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <wandb.sdk.wandb_run.Run object at 0x72f5c4583940>, telem_obj = feature {
  lightgbm_log_summary: true
}


    def _telemetry_callback(self, telem_obj: telemetry.TelemetryRecord) -> None:
>       self._telemetry_obj.MergeFrom(telem_obj)
E       AttributeError: 'Run' object has no attribute '_telemetry_obj'. Did you mean: '_telemetry_flush'?

.venv/lib/python3.10/site-packages/wandb/sdk/wandb_run.py:799: AttributeError
```

### Environment

WandB version: 0.17.6

OS: Ubuntu 

Python version: Python 3.10.12

Versions of relevant libraries:

**Comment by dmitryduev** [comment]
Thank you for reporting this @skaae! 
I would guess this error comes up if you're using the disabled mode in a test -- the actual integration was unaffected. 

Fixed in #8163, will be included in the 0.17.8 release this week.

**Comment by skaae** [comment]
great thanks.
